commit 0303f96acdabc866786a6261ef3bc307b250b16c
Author: YUKI Hiroshi <shimoda@clear-code.com>
Date:   Thu Feb 7 16:49:41 2013 +0900

    Fix typo:
    expors =>
    exports

diff --git a/lib/model.js b/lib/model.js
index 052309b..d52b9f4 100644
--- a/lib/model.js
+++ b/lib/model.js
@@ -25,7 +25,7 @@ function RequestResponse(options) {
   this._modelTypes.push(RequestResponse);
 }
 RequestResponse.prototype = new CommandModel();
-expors.RequestResponse = RequestResponse;
+exports.RequestResponse = RequestResponse;
 
 
 function PublishSubscribe(options) {
@@ -33,7 +33,7 @@ function PublishSubscribe(options) {
   this._modelTypes.push(PublishSubscribe);
 }
 PublishSubscribe.prototype = new CommandModel();
-expors.PublishSubscribe = PublishSubscribe;
+exports.PublishSubscribe = PublishSubscribe;
 
 
 
@@ -48,13 +48,13 @@ Object.defineProperty(REST.prototype, 'path', {
 Object.defineProperty(REST.prototype, 'method', {
   get: function() { return this._options.method || 'GET'; }
 });
-expors.REST = REST;
+exports.REST = REST;
 
 
 
 function SocketCommand() {
 }
-expors.SocketCommand = SocketCommand;
+exports.SocketCommand = SocketCommand;
 
 
 function SocketRequestResponse(options) {
@@ -63,7 +63,7 @@ function SocketRequestResponse(options) {
   this._modelTypes.push(SocketRequestResponse);
 }
 SocketRequestResponse.prototype = new RequestResponse();
-expors.SocketRequestResponse = SocketRequestResponse;
+exports.SocketRequestResponse = SocketRequestResponse;
 
 
 function SocketPublishSubscribe(options) {
@@ -72,4 +72,4 @@ function SocketPublishSubscribe(options) {
   this._modelTypes.push(SocketPublishSubscribe);
 }
 SocketPublishSubscribe.prototype = new PublishSubscribe();
-expors.SocketPublishSubscribe = SocketPublishSubscribe;
+exports.SocketPublishSubscribe = SocketPublishSubscribe;
